src/client/webpack.config.js

```diff
@@ -43,6 +43,9 @@ const webpackConfig = {
     ]),
     new webpack.optimize.CommonsChunkPlugin(/* chunkName= */'vendor', /* filename= */'vendor.js', function(module){
       return module.resource && module.resource.indexOf('node_modules') !== -1;
+    }),
+    new webpack.DefinePlugin({
```

Comment:
Think this is good evidence option to enhance the config.json so include the version and to a proper solution now as discussed on slack. Can discuss tomorrow after scrum perhaps?